### PR TITLE
Removed embedded images logic from the text

### PR DESCRIPTION
**Pull request by mananjadhav**
@mountiny Current testing the changes

### Details
- Removed the logic of the embedded images from the code
- Added logic to strip images from the HTML

### Fixed Issues

$ https://github.com/Expensify/App/issues/4888

### Tests
1. Tested with Text + Emoji
2. Tested with formatted text + Emoji
3. Tested with Text + Images from Gmail

### QA Steps
1. Go to chat
2. Copy any text + emoji, for example: "Testing the text with 😄 "
3. Paste and ensure that the text is copied by trimming the emoji
4. Same applies to the image as well

### Tested On

- [x] Web
- [x] Mobile Web
- [ ] Desktop
- [ ] iOS
- [ ] Android

### CODE REVIEW

File: src/components/TextInputFocusable/index.js [hunk 1]
```diff
@@ -267,43 +268,8 @@ class TextInputFocusable extends React.Component {
...
-                        * Synthetically-triggered paste events do not affect the document's contents.
-                        * See https://developer.mozilla.org/en-US/docs/Web/API/Element/paste_event for more details.
-                        */
-                        this.handlePastedHTML(pastedHTML);
-                    });
-                return;
-            }
-
-            this.handlePastedHTML(pastedHTML);
+            this.handlePastedHTML(this.stripImageTags(pastedHTML));
```
Comment (parasharrajat) [inline src/components/TextInputFocusable/index.js:272]: @mananjadhav I don't think you need to strip image tags here. They are automatically stripped

Comment (mountiny) [reply src/components/TextInputFocusable/index.js:272]: Good point! @mananjadhav try to test it without the `stripImageTags`.

Comment (mananjadhav) [reply src/components/TextInputFocusable/index.js:272]: Thanks. Still worked without `stripImageTags`

**Comment by mananjadhav** [comment]
@mountiny While this code is working fine for Text + Emojis and Text + Images, it doesn't specifically work well for Slack and Github.

Here's what it looks like in my Slack.

image

and here's what happens when it gets pasted.
```
Text with imageavatar.jpeg __[](https://files.slack.com/files-pri/T03SC9DTT-F02DUF21NUX/avatar.jpeg)
```

Can you tell me what will be the expected output in this case?

**Comment by mountiny** [comment]
@mananjadhav Thanks for the update.

This seems quite tricky. I assume based on the expectation we have stated before, we would want to paste only `Text with image`. 

But what I am wondering is how to make this more platform-agnostic. Let's say copy-pasting from different platforms might have ever so slightly different format.

**Comment by mananjadhav** [comment]
> But what I am wondering is how to make this more platform-agnostic. Let's say copy-pasting from different platforms might have ever so slightly different formats.

That is what I am checking right now. I am also checking texts from different platforms to get a pattern and close it. But it seems this might take a bit longer for testing and review.

**Comment by mountiny** [comment]
@mananjadhav Thanks. Can I ask you to pause on this PR for a bit as it is proving to be more complex than the original proposal. I will need to discuss the solution with other coworkers as it might be a bigger change.

Thank you very much for patience 🙇

**Comment by mountiny** [comment]
@mananjadhav I am creating a post in `#expensify-open-source` about this and I will tag you in too for visibility.

We also need to keep in mind, that copy/pasting text with emojis from Github/Notion/WhatsApp or Google Docs works as expected, and we would not want to break those flows here.

**Comment by kadiealexander** [comment]
Hi @mananjadhav, I've placed this issue on hold as per [this update](https://expensify.slack.com/archives/C01GTK53T8Q/p1631754482374800), as we are prioritising issues related to a feature release scheduled for 9/24. As an apology for the delay, we will add a $100 bonus as a thank you for waiting.

**Comment by mountiny** [comment]
Closing thi PR as per this [comment](https://github.com/Expensify/App/issues/4888#issuecomment-922142322).

**Pull request closed without merging**